nemo/collections/tts/helpers/partialconv1d.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+# BSD 3-Clause License
+#
+# Copyright (c) 2012, NVIDIA CORPORATION. All rights reserved.
+#
+# Author & Contact: Guilin Liu (guilinl@nvidia.com)
+###############################################################################
```

Comment:
Can you make a list of code that drew inspiration from outside the ADLR codebase? Whether that is internal to not.

Reply:
I have made a list and put them in a google doc. What is the goal?

Reply:
We need to file SWIPATs for code usage